### PR TITLE
NameDayOfDateString

### PR DESCRIPTION
**Pull request by rogergcc**

### CODE REVIEW

File: Task 1/NameDayOfDate/NameDayOfDate.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+     const date = new Date(dateString);
+     
+     const options = {
+      weekday: 'long'
+    };
+  
+    dayName = new Intl.DateTimeFormat('en-Us', options).format(date);
+
+    return dayName;
+}
```
Comment (nimishbongale) [inline Task 1/NameDayOfDate/NameDayOfDate.js:15]: It's a good practice to add a newline at the end of file!

**Review by nimishbongale — COMMENTED** [review]
Good work @rogergcc!

Really small change requested. Let's get this merged!

File: Task 1/NameDayOfDate/NameDayOfDate.js (hunk 1 above)
Comment (rogergcc) [reply Task 1/NameDayOfDate/NameDayOfDate.js:15]: Sure

**Review by nimishbongale — APPROVED** [review]
Great work @rogergcc! 🚀 

You are now ready to contribute to cool open source projects! 😄

**Pull request merged**